spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -572,6 +600,17 @@ def save_figure_as(self, fig, fmt):
         if fname:
             save_figure_tofile(fig, fmt, fname)
 
+    def copy_current_figure(self):
+        """Copy the currently selected figure."""
+        if self.current_thumbnail is None:
+            return
+
+        b_succeed = copy_figure(self.current_thumbnail.canvas.fig,
```

Comment:
Can you name this more descriptively? I'm not sure what ``b`` is supposed to stand for here. Remember the Zen of Python: Explicit is better than implicit.

Reply:
I will convert 'b' to 'is'.

Reply:
Okay, thanks. I *still* have no idea what ``b`` was supposed to stand for, heh, but maybe I'm just dumb.

Reply:
'b' means boolean. ^^

Reply:
Oh, okay. FYI, Hungarian notation is mostly not used in modern programming languages, and certainly not in Python since its considered quite unPython and specifically prohibited by PEP 8.